healthcare/healthcare/doctype/lab_test/lab_test.py

```diff
@@ -67,6 +76,34 @@ def validate_result_values(self):
 					frappe.throw(_('Row #{0}: Please enter the result value for {1}').format(
 						item.idx, frappe.bold(item.lab_test_particulars)), title=_('Mandatory Results'))
 
+	@frappe.whitelist()
+	def modify_nursing_tasks(self, nursing_tasks):
+		template_tasks = frappe.get_list(
+			'Nursing Checklist Template Task',
+			filters=self.get_nursing_task_filters(self.template),
+			fields=['name', 'activity', 'mandatory', 'task_doctype'],
+		)
```

Comment:
```suggestion
		template_tasks = frappe.get_all(
			'Nursing Checklist Template Task',
			filters=self.get_nursing_task_filters(self.template),
			fields=['name', 'activity', 'mandatory', 'task_doctype'],
		)
```

We should not use get_list on the child table since it checks for perms.